features/support/helpers.rb

```diff
@@ -0,0 +1,20 @@
+module AmazonSearchElementsHelper
+
+  def sort_search_results(sortedBy)
+    sortedByValue = SortTextToValue(sortedBy)
```

Comment:
you can use "text" option in `select_option_from_dropdown` to avoid this function (you would search for "Price: High to Low"). Try to do this

Reply:
I had a problem with search by Text because it changes depending the web language so using the Value seems to be like a better option